xclim/core/units.py

```diff
@@ -282,15 +294,16 @@ def pint_multiply(
     Returns
     -------
     xr.DataArray
+        The product DataArray.
     """
     a = 1 * units2pint(da)  # noqa
-    f = a * q.to_base_units()
+    _f = a * q.to_base_units()
```

Comment:
why change `f` but not `a` ?

Reply:
Good point. Will look into that.

Reply:
I just checked, this is not like the `_x` and `_y` earlier in a sub-function that you wanted a distinction from the parent function

Reply:
The IDE was complaining because of this earlier in the file:
```python
with (files("xclim.data") / "variables.yml").open() as f:
    CF_CONVERSIONS = safe_load(f)["conversions"]
_CONVERSIONS = {}
```
Will rename this here and revert the changes in the function.